ubuntu-18-agent-aws.json

```diff
@@ -0,0 +1,62 @@
+{
+  "variables": {
+    "compose_version": "1.25.4",
+    "maven_version": "3.6.3"
+  },
+  "builders": [
+    {
+      "type": "amazon-ebs",
+      "region": "{{user `location`}}",
+      "access_key": "{{user `aws_access_key`}}",
+      "secret_key": "{{user `aws_secret_key`}}",
+      "ssh_username": "ubuntu",
+      "source_ami_filter": {
+          "filters": {
+              "virtualization-type": "hvm",
+              "name": "ubuntu/images/hvm-ssd/ubuntu-bionic-18.04-amd64-server-*",
+              "root-device-type": "ebs"
+          },
+          "owners": ["099720109477"],
+          "most_recent" : true
+      },
+      "instance_type": "t2.micro",
+      "ami_name": "jenkins-agent-ubuntu18.04",
+
+      "tags": {
+        "timestamp": "{{isotime \"20060102150405\"}}",
+        "imagetype": "jenkins-agent-ubuntu18.04"
+      }
+    }
+  ],
+  "provisioners": [{
+    "execute_command": "chmod +x {{ .Path }}; {{ .Vars }} sudo -E sh '{{ .Path }}'",
+    "inline": [
```

Comment:
is is worth extracting this out to a script? I don't see any changes between aws and azure here?

Reply:
It would a nice improvement but not mandatory IMHO.

Reply:
should be straightforward since windows is done like this

Reply:
Should be pretty easy, I can pull it out as part of this.